### PR TITLE
[MASTER] - Add more information to the hostname change document

### PR DESCRIPTION
**Pull request by deshankoswatte**
## Purpose
- Add more information regarding the places which the local host value should be changed.
- Fixes: https://github.com/wso2/product-is/issues/13910.

### CODE REVIEW

File: en/docs/deploy/change-the-hostname.md [hunk 1]
```diff
@@ -174,4 +174,20 @@ you can make sure that the existing encrypted data is not affected.
...
+the references should be changed.
+
+- The configurations in the `<IS_HOME/repository/conf/deployment.toml` file that references localhost.
+- The entity ID that is available in the resident identity provider configurations.
+- The callback URLs of the federated identity providers.
+- The callback URLs of the default system applications such as My Account and Console.
+
+!!! note
+
+    To edit the default system applications such as My Account and Console, the following configuration should be added
```
Comment (nilminiwso2) [inline en/docs/deploy/change-the-hostname.md:187]: What are we editing in the default system applications here?
Also, can you explain how this is relevant to changing the hostname?

@deshankoswatte

Comment (deshankoswatte) [reply en/docs/deploy/change-the-hostname.md:192]: These changes are added with https://github.com/wso2/docs-is/pull/3071/commits/5cd0ac947cc297f52fb1c3104382b1610e8e65be.

File: en/docs/deploy/change-the-hostname.md [hunk 2]
```diff
@@ -174,4 +174,25 @@ you can make sure that the existing encrypted data is not affected.
...
+- The configurations in the `<IS_HOME/repository/conf/deployment.toml` file that references localhost.
+- The entity ID that is available in the resident identity provider configurations.
+- The callback URLs of the federated identity providers.
+- The callback URLs of the default system applications such as My Account and Console.
+
+!!! note
+
+    By default, both the My Account and the Console Applications use the `localhost:9443` domain within the callback 
+    URL. When you change the hostname, the WSO2 Identity Server will not be running on the `localhost:9443` domian and 
+    these values will not automatically change hence you should manually change the callback URLs to use the new
```
Comment (nilminiwso2) [inline en/docs/deploy/change-the-hostname.md:189]:
```suggestion
    these values will not automatically change. Hence, you should manually change the callback URLs to use the new 
```

File: en/docs/deploy/change-the-hostname.md [hunk 3]
```diff
@@ -174,4 +174,25 @@ you can make sure that the existing encrypted data is not affected.
...
+
+!!! note
+
+    By default, both the My Account and the Console Applications use the `localhost:9443` domain within the callback 
+    URL. When you change the hostname, the WSO2 Identity Server will not be running on the `localhost:9443` domian and 
+    these values will not automatically change. Hence, you should manually change the callback URLs to use the new 
+    hostname.
+    
+    To edit the default system applications such as My Account and Console, the following configuration should be added
+    to the `<IS_HOME/repository/conf/deployment.toml` file.
```
Comment (nilminiwso2) [inline en/docs/deploy/change-the-hostname.md:193]:
```suggestion
    to the `<IS_HOME/repository/conf/deployment.toml` file:
```

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**